Cargo.toml

```diff
@@ -58,7 +57,7 @@ ucan = "0.1"
 url = "2.3"
 wasmer = { version = "3.1", features = ["compiler"] }
 wasmer-compiler-singlepass = "3.1.0"
-wasmer-middlewares = "3.1.0"
```

Comment:
Is there a way to prevent `cargo add` from doing this? Or is the problem that I'm using `cargo add`? 😅